knowledge_repo/app/index.py

```diff
@@ -29,6 +34,8 @@ def update_index():
             post.status = current_repo.PostStatus.UNPUBLISHED
             continue
 
+        post.keywords = get_keywords(post)
```

Comment:
That'd be great. I'd guess that others might prefer a workflow that tags posts after contribution as opposed to in the document. But that's still speculative, so can revisit that later if it's annoying to do now.